src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -217,6 +231,20 @@ function ProviderSelectionField({
             </button>
           </div>
         )}
+      {showRemoveProviderModal && (
+        <RemoveProviderModal
+          provider={formData}
+          address={address}
+          onClose={response => {
+            setShowRemoveProviderModal(false);
+            if (response === true) {
+              setRemovedProvider(true);
```

Comment:
I think you'd also need to set removedProvider back to false at some point, probably when the modal is opened. Otherwise, if someone choose a provider, removes it, chooses another, then removes it, the focus won't work correctly.

And actually, I wonder if you could get away with not having `removedProvider` in state? You could have an effect that fired when `formData` changed, and then set focus to the Add provider link when `mounted` was true and `formData` was an empty object.

Reply:
Tweaked this a bit so should be good now.